sdk/eventhub/event-hubs/src/connectionContext.ts

```diff
@@ -169,39 +198,52 @@ export namespace ConnectionContext {
       connectionContext.connection.removeAllSessions();
 
       // Close the cbs session to ensure all the event handlers are released.
-      await connectionContext.cbsSession.close();
+      await connectionContext.cbsSession.close().catch((err) => {
```

Comment:
Precaution or did you actually see errors getting thrown?

Reply:
I only made this change because in some of my extreme test runs I did see this throw, though it required nearly blocking the event loop for a long time.